content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -139,26 +191,44 @@ Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.
 Assign stages to a team
 -----------------------
 
-Make a selection in the :guilabel:`Teams` field on the :guilabel:`Stages` form. More than one team
-may be selected, since the same stage(s) can be assigned to multiple teams.
-
- .. image:: getting_started/stages-settings-sharing.png
-   :align: center
-   :alt: View of stage setup emphasizing teams field
+Make a selection in the :guilabel:`Helpdesk Teams` field on the :guilabel:`Stages` form. More than
+one team may be selected, since the same stage can be assigned to multiple teams.
 
 Fold a stage
 ------------
 
-Check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form to display this stage as
-*folded* by default in the kanban view for this team.
+By default, stages are unfolded in the Kanban view of either tickets dashboard: :guilabel:`My
+Tickets` (:menuselection:`Helpdesk app --> Tickets --> My Tickets`) or :guilabel:`All Tickets`
+(:menuselection:`Helpdesk app --> Tickets --> All Tickets`).
+
+Tickets in an unfolded stage are visible in the pipeline under the stage name, and are considered
+*open*.
+
+Stages can be configured to be folded in the Kanban view of a tickets page (:guilabel:`My Tickets`
+or :guilabel:`All Tickets`).
+
+The name of the folded stages are still visible, though the tickets in the stage are no longer
+immediately visible.
+
+To fold a stage, check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form.
 
 .. warning::
-   Tickets that reach a *folded* stage are considered closed. Closing a ticket before the work is
-   completed can result in reporting and communication issues. This setting should only be enabled
-   for stages that are considered *closing* stages.
+   Tickets that reach a *folded* stage are considered `closed`. Closing a ticket before the work is
```

Comment:
I would argue here that formatting on "closed" is not necessary, either italics to emphasize or nothing. Using the literal formatting is reserved for inputs. WDYT about formatting here?
```suggestion
   Tickets that reach a *folded* stage are considered *closed*. Closing a ticket before the work is
```